### DISCUSSION TITLE
是否有迁移到winui3的计划？

### DISCUSSION THREAD

**Discussion by JimHans** [ywmoyue/biliuwp-lite#219] [General]
目前的uwp使用winui2.8框架，微软已经不再为它提供新功能支持了，且升级winui3应该工作量不算太大（没做过uwp开发不太清楚，但微软自己是这么说的），那么是否有迁移的计划呢？
而且最近自己的win11机器一直有winui2框架播放器播放画面卡死的问题，自己动手修了两个多月一直没成效，只能等winui3了。

**Comment by ywmoyue** [comment]
我计划先把代码按一定规范整理一遍，再做迁移winui3

**Reply by JimHans** [reply]
好的，非常感谢，期待完成！
